templates/messageextension/csharp/default/Bot/TeamsMessageExtension.tpl

```diff
@@ -0,0 +1,210 @@
+﻿using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Teams;
+using Microsoft.Bot.Schema;
+using Microsoft.Bot.Schema.Teams;
+
+using Newtonsoft.Json.Linq;
+
+namespace {{BlazorAppServer}}.Bot;
```

Comment:
Since it is a separate project for ME, we could reconsider the namespace structure.